app/views/contact/suggestion_success.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render Blacklight::System::ModalComponent.new do |component| %>
+  <% component.title { t('blacklight.suggestion.form.title') } %>
```

Comment:
Need a corresponding addition to `config/locales/blacklight.en.yml` - see e.g. https://github.com/pulibrary/orangelight/blob/0528c1d2ac991fc7ed87bf82e72e7af802d7ccc6/config/locales/blacklight.en.yml#L82-L84